tests/component/sections/layout/header/Header.spec.tsx

```diff
@@ -37,6 +37,23 @@ describe('Header component', () => {
     cy.findByText('Log Out').should('be.visible')
   })
 
+  it('displays the Add Data Button when the user is logged in', () => {
```

Comment:
What about the other test in Home.spec.tsx, don't we want to check the same behaviour here?

```
 it('does not render the Add Data dropdown button', () => {
     cy.customMount(<Home datasetRepository={datasetRepository} />)
     cy.findByText('Add Data').should('not.exist')
   })
```